arch/arm/boot/dts/overlays/ov5647-overlay.dts

```diff
@@ -94,6 +91,13 @@
 		};
 	};
 
+	fragment@7 {
```

Comment:
What is the purpose of this fragment?

Reply:
This is patch 2. At this point OV5647 relies on using the GPIO as a shutdown line via the GPIO API. If the cam1_reg is enabled then it will claim the GPIO and the sensor will fail to probe - regression.

Patch 3 & 4 add support for the regulator framework to OV5647.

I can swap the patches around so that we convert OV5647 to use the regulators first, and then we won't have this interim part.

Reply:
It's not interim - I'm looking at the cumulative effect of the patches and it's still there, disabling `cam1_reg`.

Reply:
Drat, I obviously missed removing it again.

Reply:
That's fine. As long as I know it is no longer needed I can add the deletion to my other patches.